components/sessions/SpeakersDetails.tsx

```diff
@@ -63,7 +74,7 @@ export const SpeakersDetails = ({ session }: { session: Session }) => {
                       >
                         @
                         {speaker.twitter
-                          ? speaker.twitter.split('twitter.com/')[1]
+                          ? getTwitterUsername(speaker.twitter)
```

Comment:
Here, why not? 
```js
{getTwitterUsername(speaker.twitter) ?? speaker.name}

Reply:
Thanks @danielotieno . Let me implement that. What do you think about
`{getTwitterUsername(speaker.twitter ?? '') ?? speaker.name}` to ensure that speaker.twitter is a string before passing it to `getTwitterUsername`

Reply:
`{getTwitterUsername(speaker.twitter ?? '') || speaker.name}`
you can check the usage of || vs ??

Reply:
Or if you want to use ternary 

```js
{speaker.twitter 
  ? getTwitterUsername(speaker.twitter) || speaker.name
  : speaker.name}

Reply:
Yes ` || ` would be best in this case. I've already resolved this and pushed. Please review.